### PR TITLE
Add CI

### PR DESCRIPTION
**Pull request by IanVaughan**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: "Tests"
```
Comment (hebbo) [inline .github/workflows/tests.yml:1]: Is it worth to create a workflow just for rubocop linting?
Alternatively, we could name this workflow `name: "Lint and test" 

FF2CC

**Review by hebbo — APPROVED** [review]

File: .github/workflows/tests.yml (hunk 1 above)
Comment (IanVaughan) [reply .github/workflows/tests.yml:1]: For something simple like this gem I don't think the overhead of time and cost is worth it.

**Comment by IanVaughan** [comment]
@hebbo approve?? cheers

**Comment by hebbo** [comment]
> @hebbo approve?? cheers

🤔 
Screenshot 2023-08-16 at 12 09 25

**Comment by IanVaughan** [comment]
But:
Screenshot 2023-08-16 at 11 14 18

**Review by hebbo — APPROVED** [review]

**Comment by hebbo** [comment]
Maybe I don't have write access to the repo

**Comment by IanVaughan** [comment]
Just gave access, and it now shows approved! cheers

**Pull request merged**